### DISCUSSION TITLE
Cancel job button not sending workload request

### DISCUSSION THREAD

**Discussion by q-benwillis** [microsoft/Microsoft-Fabric-workload-development-sample#127] [Help] [answered]
When I click the cancel job button in the UI:
![image](https://github.com/user-attachments/assets/87b0fcb5-97fc-489d-800a-62832e4a05ce)
I'm not seeing any cancel requests appearing in my local dev gateway.

I would expect the cancel endpoint to be hit: https://learn.microsoft.com/en-us/rest/api/fabric/workload/workloadapi/jobs/cancel-item-job-instance?tabs=HTTP

**Comment by michal-mano** [comment, marked as answer]
Hi,
When the cancel button is clicked it triggers this case "item.job.cancel":
https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/main/Frontend/src/index.worker.ts#L80-L83

In our implementation, this reaches the sample workload controller:
https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/main/Frontend/src/controller/SampleWorkloadController.ts#L643-L649

And there isn't an implementation in the backend, but the shell is here:
https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/main/Backend/src/Controllers/JobsControllerImpl.cs#L80-L83

Hope this helps.
